src/algebra/lie_algebra.lean

```diff
@@ -63,9 +63,11 @@ variables {A : Type v} [ring A]
 The ring commutator captures the extent to which a ring is commutative. It is identically zero
 exactly when the ring is commutative.
 -/
```

Comment:
This docstring should be rewritten or deleted.

Reply:
Why?

Reply:
Because now it refers to an instance instead of a def? I am not really sure how documentation works then

Reply:
Since the docstring hadn't been changed (there's still even an extra line break), I wasn't sure whether you'd forgotten about it in the refactor. By the way, the linter doesn't complain about docstrings for `instance`s, but it's often a good idea to include them anyways. 

I would suggest something like "The bracket operation for rings is the ring commutator, which captures the extent to which a ring is commutative. It is identically zero exactly when the ring is commutative."

Reply:
Ops yeah I forgot the line break. I will change that together with @Vierkantor's next comments if there'll be any